src/Quaternion.cpp

```diff
@@ -83,6 +83,37 @@ vr::HmdQuaternion_t QuaternionFromAngle(const double& xx, const double& yy, cons
 	return quat;
 }
 
+vr::HmdMatrix33_t QuaternionToMatrix(const vr::HmdQuaternion_t q) {
+	
+	vr::HmdMatrix33_t result = { {
+		{1 - 2*q.y*q.y - 2*q.z*q.z, 2*q.x*q.y - 2*q.z*q.w, 2*q.x*q.z + 2*q.y*q.w},
+		{2*q.x*q.y + 2*q.z*q.w, 1 - 2*q.x*q.x - 2 * q.z*q.z, 2*q.y*q.z - 2*q.x*q.w},
+		{2*q.x*q.z - 2*q.y*q.w, 2*q.y*q.z + 2*q.x*q.w, 1 - 2*q.x*q.x - 2*q.y*q.y}
+		} };
+
+	return result;
+}
+
+double QuatNorm(const vr::HmdQuaternion_t q) {
+	return sqrt(q.w * q.w + q.x * q.x + q.y * q.y + q.z * q.z);
+}
+
+vr::HmdQuaternion_t QuatConjugate(const vr::HmdQuaternion_t q) {
+	vr::HmdQuaternion_t quat = { q.w,-q.x,-q.y,-q.z };
+	return quat;
+}
+
+vr::HmdQuaternion_t QuatInverse(const vr::HmdQuaternion_t q) {
+	double abs = QuatNorm(q);
+	double d = abs * abs;
+
+	vr::HmdQuaternion_t conj = QuatConjugate(q);
+
+	vr::HmdQuaternion_t quat = { conj.w / d,conj.x / d,conj.y / d, conj.z / d };
+	return quat;
+
+}
+
```

Comment:
Do we need `QuatInverse()` anymore? Since we are dealing with unit quaternions it should just be the conjugate.